src/main/cpp/chassis/HolonomicDrive.cpp

```diff
@@ -146,7 +146,7 @@ void HolonomicDrive::Run()
             SlowMode();
         }
 
-        if (units::math::abs(m_moveInfo.chassisSpeeds.omega).to<double>() > 0.03)
+        if (units::math::abs(m_moveInfo.chassisSpeeds.omega).to<double>() > 0.1)
```

Comment:
This one doesn't make sense to me; why if we've specified a rotation are we switching to maintain heading?

Reply:
I agree, I didn't make this change, Nick did. What is weird he just changed the number. Updated to just use the axis input